src/Idris/REPL.idr

```diff
@@ -440,7 +448,11 @@ processEdit (GenerateDef upd line name rej)
                     put ROpts (record { gdResult = Just (line, searchdef) } ropts)
                     Just (_, (fc, cs)) <- nextGenDef rej
                          | Nothing => pure (EditError "No search results")
-                    let l : Nat =  integerToNat (cast (snd (startPos fc)))
+
+                    let l : Nat = integerToNat $ cast $ case fc of
+                         (MkFC _ (_, col) _) => col
+                         _ => the Int 0
+
```

Comment:
The same pattern is use twice, can we introduce `startCol : FC -> Int` in `Core.FC`?